docs/user-guide/healpix.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>Can you link the equation to a HEALPix resource, I think this one: <a href="https://easy.gems.dkrz.de/Processing/healpix/index.html#healpix-spatial-resolution" target="_blank">https://easy.gems.dkrz.de/Processing/healpix/index.html#healpix-spatial-resolution</a> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Sure! I need to fix the latex as well.